packages/ai/src/prompts/chat.ts

```diff
@@ -60,12 +61,36 @@ export type ChatPromptSendOptions<TOptions extends Record<string, any> = Record<
   readonly onChunk?: TextChunkHandler;
 };
 
+export type ChatPromptPlugin<TPluginName extends string, TPluginUseArgs extends {}> = {
+  /**
+   * Unique name of the plugin
+   */
+  name: TPluginName;
+
+  /**
+   * The function is set to initialize the plugin
+   * @param args - Arguments to initialize the plugin
+   */
+  usePlugin?: (args: TPluginUseArgs) => void;
```

Comment:
the name `onInit` might be more clear and match that of the apps package plugins

Reply:
The way to use it would be
```
chatPrompt.usePlugin('mcp', ...).usePlugin('mcp', ...)
```
Multiple `init`s might be kinda confusing?
```
chatPrompt.onInit('mcp', ...).onInit('mcp', ...)
```
Thoughts?